<?xml version="1.0" ?>
<launch>
  <node pkg="toc_execution" type="toc_execution_node.py" name="toc_execution_node" output="screen">
    <param name="robot_id" value="pumpkin" />
    <param name="wait_duration" value="1" />
    <param name="timeout_duration" value="1000" />
  </node>

  <node pkg="toc_execution" type="toc_mdp_state_monitor.py" name="toc_mdp_state_monitor" output="screen">
    <param name="rate" value="10" />
  </node>
</launch>